frontend/packages/console-app/src/components/quick-starts/QuickStartPanelContent.tsx

```diff
@@ -67,7 +72,7 @@ const QuickStartPanelContent: React.FC<QuickStartPanelContentProps> = ({
       <DrawerPanelBody hasNoPadding className="co-quick-start-panel-content__body">
         <AsyncComponent
           loader={() => import('./QuickStartController').then((c) => c.default)}
-          quickStart={quickStart}
+          quickStart={{ ...quickStart, nextQuickStarts }}
```

Comment:
Hmmmm, why you don't added a second prop `nextQuickStarts` here and in the `QuickStartController`?